imdown-mobile/login/login.tsx

```diff
@@ -7,8 +7,9 @@ import { TouchableHighlight } from "react-native-gesture-handler";
 
 
 const Login = ({ navigation }) => {
-  const IOS_CLIENT_ID = '1097983281822-8qr8vltrud1hj3rfme2khn1lmbj2s522.apps.googleusercontent.com'
-  const LOGIN_SUCCESS = "success"
+  const IOS_CLIENT_ID = "1097983281822-8qr8vltrud1hj3rfme2khn1lmbj2s522.apps.googleusercontent.com";
+  const ANDROID_CLIENT_ID = "1097983281822-4b63n721lbqllpn7u4cvoqmh0rudquma.apps.googleusercontent.com";
```

Comment:
lol eventually we should probably move these to not be in code but all good for now

Reply:
yeah I mean these params are only used for the expo mobile app anyway (and they are safe for public exposure)